drivers/pcie/host/pcie.c

```diff
@@ -4,6 +4,13 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+#if defined(CONFIG_PCIE_PRT)
+#if defined(CONFIG_X86_64)
+#define DT_DRV_COMPAT intel_pcie
```

Comment:
why not DT_DRV_COMPAT pcie_controller  ? (and thus, getting rid of the CONFIG_X86_64 above).

Up to the relevant architecture to set CONFIG_PCIE_PRT accordingly.

Reply:
updated

Reply:
> why not DT_DRV_COMPAT pcie_controller ? (and thus, getting rid of the CONFIG_X86_64 above).
> 
> Up to the relevant architecture to set CONFIG_PCIE_PRT accordingly.

@tbursztyka maybe this is just me not understanding how DTS works, but we don't have any pcie nodes using a "pcie,controller" compat string. We do have a `pice-controller.yaml` that PCIe bindings include, but does that mean that our tooling generates DT macros to match that file as well?

Reply:
see my answer in the comment below